### ISSUE TITLE
shipit-engine: Classification Check

### ISSUE THREAD

**Issue by shopify-services** [Shopify/shipit-engine#1117]
# Classification Check

## TL;DR
1st party software must have a business impact classification. See our [list of classification](https://services.shopify.io/doc/classifying_services_in_tiers) to help you determine the classification of your application. 

## Why is this being asked?
It's critical that we understand the business impact of the software we write. The classification of your service determines the level of support you must provide as owners and helps us set standards.

## What will happen if it doesn't get done within the expected timeframe?
This is mandatory. If this is not done you won't be able to have any runtimes.

## When does it need to get done?

At the latest, this should be done before **2021-07-22.**

## This doesn't apply to my service. What do I do?
First, leave a comment explaining why it doesn't apply. Then, leave another comment as `/not_applicable`, and close the issue.
If you change your mind, make sure to comment any reason and reopen the issue.

## I have questions/concerns about this
Please contact the Production Excellence team using Slack at #production-excellence-team.
**Your service**: [shipit-engine/rubygems](https://services.shopify.io/services/shipit-engine/rubygems)
**Owners**:

**Comment by shopify-services** [comment]
**Owners**: 
The action items due date was updated. Please review
